docs/writing-stories/introduction.md

```diff
@@ -11,7 +11,7 @@ Storybook uses the generic term arguments (args for short) when talking about Re
 A component’s stories are defined in a story file that lives alongside the component file. The story file is for development-only, and it won't be included in your production bundle.
 
 ```
-Button.js | ts | jsx | tsx
+Button.vue | js | ts | jsx | tsx 
 Button.stories.js | ts | jsx | tsx | mdx
 ```
```

Comment:
@giladtamam we have a bit of room for improvement here, if you wouldn't mind we can extend this and also incorporate Svelte. Turning this into something like:

"
Button.js | ts | jsx | tsx | vue | svelte 
"

The reasoning behind this is that it provides a more comprehensive list of component formats that you and other readers can use with their Storybook? What do you think?